include/Row.hpp

```diff
@@ -227,11 +227,33 @@ class Row {
     }
 
     /*
+     * Retrieve the value at the specified column index as a geographical point.
+     * This data type is not currently supported by the C++ client so this always
+     * throws an exception.
+     * @throws UnsupportedTypeException always
+     * @return never returns
+     */
+    void getGeographyPoint(int32_t column) {
+        throw UnsupportedTypeException(wireTypeToString(WIRE_TYPE_GEOGRAPHY_POINT));
+    }
+
+    /*
+     * Retrieve the value at the specified column index as a geographical object.
+     * This data type is not currently supported by the C++ client so this always
+     * throws an exception.
+     * @throws UnsupportedTypeException always
+     * @return never returns
+     */
+    void getGeography(int32_t column) {
+        throw UnsupportedTypeException(wireTypeToString(WIRE_TYPE_GEOGRAPHY));
+    }
+
+    /*
      * Returns true if the value at the specified column index is NULL and false otherwise
      * @throws InvalidColumnException The name of the column was invalid.
      * @return true if the value is NULL and false otherwise
      */
-    bool isNull(int32_t column) throw(voltdb::InvalidColumnException) {
+    bool isNull(int32_t column) throw(voltdb::InvalidColumnException, voltdb::UnsupportedTypeException) {
```

Comment:
Yeah, I had heard throw declarations were falling out of favor.  I found that I had to put it in if I wanted to be able to catch the exception in the test app.  If I left it out the process would abort before I could catch the exception.

Maybe we should have a ticket to remove all the throws declarations?  They seem only to make more work for us.